packages/rpc-api/src/getLeaderSchedule.ts

```diff
@@ -67,6 +86,16 @@ export type GetLeaderScheduleApi = {
                 identity: Address;
             }>,
     ): GetLeaderScheduleApiResponseWithSingleIdentity<TIdentity>;
+    /**
+     * Fetch the leader schedule of all validators.
+     *
+     * @param slot When `null`, orders the leader schedule for the current epoch.
+     *
+     * @returns A dictionary having a single key representing the specified validator identity, and
+     * its corresponding leader slot indices as values relative to the first slot in the current
+     * epoch.
+     * @see https://solana.com/docs/rpc/http/getleaderschedule
+     */
     getLeaderSchedule(
         slot: null,
```

Comment:
```suggestion
        slot?: null,
```

I think `slot` should be optional here, because `"params": []` is valid, and returns all identities for the current epoch

Reply:
Nice. I also added a typetest for ‘no params.’